### ISSUE TITLE
thank you from first user

### ISSUE THREAD

**Issue by bchtrue** [kpion/netmon#1]
Thanks :) it was hard for me to find this addon because it show as only 1 user :) I will use it!

1) need the way to check "response/request body" at least for xhr requests. maybe this floating window with all info show as new popup
2) quick global disable / enable addon from statusbar

**Comment by bchtrue** [comment]
@kpion And the big problem it lose redirects. It show them as domain.com -- initiator and **at the same line** requested domain will be other_redirected_domain.com. It must be clean and show all redirects maybe one per line or somehow understandable without loss of information. domain1 -> redirect to domain2 -> redirect_to domain3.com  (3 lines)

**Comment by kpion** [comment]
@bchtrue - hah, yeah, it's new and I guess too specific. And yes, a few things need to be added/changed as well :)

Re 1. need the way to check "response/request body" - yes, I have it on my to do list, but it isn't easy, for some reason the browsers do send it on some occasions, and on others they do not o.O I have to investigate it further.

Re. 2. quick global disable / enable addon from statusbar - there is the 'pause' button - this 'pauses' the monitor for current tab... eh, just noticed that it also forgets about it when we close the popup. Yeah ok, I'll add it.

Re.  'redirections' - OK, will be done as well :)

Thank you very much! Will ping you here, when done.

**Comment by bchtrue** [comment]
@kpion Thank you! I'm not every day at github, but I will try to watch for your repo :) Very usefully addon.

**Comment by bogn83** [comment]
Yup, nice addons, thanks! Also voting for the global disable/enable. I guess the way uBlock Origin does it with the big power icon is quite intuitive: https://addons.mozilla.org/de/firefox/addon/ublock-origin/ ultra big bonus points for greying out the icon in the top right similar to how they do it.

**Comment by laniakea64** [comment]
Hi, I too would like to say thank you @kpion for this important and useful addon!

> @kpion And the big problem it lose redirects. 

> Re. 'redirections' - OK, will be done as well :)

I'm seeing this same issue, any update on this?

If I can find the time and can at least fix the basic "loss of information", would it be helpful if I file a pull request?

**Comment by kpion** [comment]
@laniakea64 - of course it would be helpful! BTW, if not you, then I can take care of it on weekend (I'll do my best, it's like 99%) -
